src/accelerate/utils/modeling.py

```diff
@@ -206,6 +206,7 @@ def set_module_tensor_to_device(
     device: Union[int, str, torch.device],
     value: Optional[torch.Tensor] = None,
     dtype: Optional[Union[str, torch.dtype]] = None,
+    fp16_statistics: Optional[torch.HalfTensor] = None,
```

Comment:
We can probably in the future replace https://github.com/huggingface/transformers/blob/04f46a22d8b5f38c8369b3d995df3102b8d56dcd/src/transformers/utils/bitsandbytes.py#L22 to your modified method with a proper check of accelerate version for backward compatibility

Reply:
Not sure about the potential impact of doing that, so maybe keeping it like that is already great